docs/en/guides/pipeline-integrations/development.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Development
+pagename: toolkit-development
+lang: en
+---
+
+# Development
```

Comment:
As above this is blank though my PR contains content for this, so I guess we can leave this for now.

Reply:
oh whoops. forgot about that and added a small blurb. feel free to replace it with yours.

Reply:
Will do, thanks!